taskcluster/config.yml

```diff
@@ -38,3 +38,8 @@ workers:
             implementation: docker-worker
             os: linux
             worker-type: linux-gcp
+        succeed:
```

Comment:
I'm curious why this is necessary? I didn't have to do this in https://github.com/mozilla-releng/balrog/commit/b593d465d3291aee3572b90cfd0b150922754c0d

Reply:
Oh, maybe it's not... I copied it from Gecko.

Reply:
I was looking for where you did that, but I couldn't find it

Reply:
Yeah, looks like it's not necessary.